packages/labs/src/components/timezone-picker/timezoneItems.ts

```diff
@@ -42,22 +42,24 @@ export function getTimezoneItems(date: Date): ITimezoneItem[] {
  * @param date the date to use when determining timezone offsets
  * @param includeLocalTimezone whether to include the local timezone
  */
-export function getInitialTimezoneItems(date: Date, includeLocalTimezone: boolean): ITimezoneItem[] {
-    const populous = getPopulousTimezoneItems(date);
-    const local = getLocalTimezoneItem(date);
+export function getInitialTimezoneItems(
+    date: Date,
+    includeLocalTimezone: boolean,
+    useManualCalc: boolean,
```

Comment:
Worth setting `useManualCalc: boolean = false`? Then you don't have to update all those tests with the extra param.